src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs

```diff
@@ -75,24 +75,46 @@ public static class FrameworkIdentifiers
+            /// <summary>net40 (.NETFramework,Version=v4.0)</summary>
             public static readonly NuGetFramework Net4 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 0, 0, 0));
+            /// <summary>net403 (.NETFramework,Version=v4.0.3)</summary>
             public static readonly NuGetFramework Net403 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 0, 3, 0));
+            /// <summary>net45 (.NETFramework,Version=v4.5)</summary>
             public static readonly NuGetFramework Net45 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 5, 0, 0));
+            /// <summary>net451 (.NETFramework,Version=v4.5.1)</summary>
             public static readonly NuGetFramework Net451 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 5, 1, 0));
+            /// <summary>net452 (.NETFramework,Version=v4.5.2)</summary>
             public static readonly NuGetFramework Net452 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 5, 2, 0));
+            /// <summary>net46 (.NETFramework,Version=v4.6)</summary>
             public static readonly NuGetFramework Net46 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 6, 0, 0));
+            /// <summary>net461 (.NETFramework,Version=v4.6.1)</summary>
             public static readonly NuGetFramework Net461 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 6, 1, 0));
+            /// <summary>net462 (.NETFramework,Version=v4.6.2)</summary>
             public static readonly NuGetFramework Net462 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 6, 2, 0));
+            /// <summary>net463 (.NETFramework,Version=v4.6.3)</summary>
             public static readonly NuGetFramework Net463 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 6, 3, 0));
+            /// <summary>net47 (.NETFramework,Version=v4.7)</summary>
             public static readonly NuGetFramework Net47 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 7, 0, 0));
+            /// <summary>net471 (.NETFramework,Version=v4.7.1)</summary>
             public static readonly NuGetFramework Net471 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 7, 1, 0));
+            /// <summary>net472 (.NETFramework,Version=v4.7.2)</summary>
             public static readonly NuGetFramework Net472 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 7, 2, 0));
+            /// <summary>net48 (.NETFramework,Version=v4.8)</summary>
+            public static readonly NuGetFramework Net48 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 8, 0, 0));
+            /// <summary>net481 (.NETFramework,Version=v4.8.1)</summary>
+            public static readonly NuGetFramework Net481 = new NuGetFramework(FrameworkIdentifiers.Net, new Version(4, 8, 1, 0));
 
+            /// <summary>net11 (.NETFramework,Version=v1.1)</summary>
```

Comment:
This group of 3 didn't get set properly

Reply:
thank you for being thorough enough to notice!